src/components/member/tabs/ClaimsTab.tsx

```diff
@@ -1,202 +1,154 @@
-              )
-            })
-            return { date: new Date(), value: 'EMAIL', open: false }
-          },
-          typeChangeHandler: (event) => (_) => ({
-            value: event.target.value,
-          }),
-          dateChangeHandler: (date) => {
-            return {
-              date,
-            }
-          },
-        }}
-      >
-        {({
-          handleClose,
-          handleOpen,
-          dateChangeHandler,
-          typeChangeHandler,
-          handleClaimSubmit,
-          open,
-          value,
-          date,
-        }) => (
-          <>
+      <HeaderWrapper>
+        <MainHeadline>
+          Claims
+          <RefreshButton onClick={() => refetch()} loading={loading}>
+            <ArrowRepeat />
+          </RefreshButton>
+        </MainHeadline>
+
+        <FormWrapper>
+          {showForm ? (
+            <>
+              <EnumDropdown
+                enumToSelectFrom={ClaimSource}
+                placeholder={'Claim Source'}
+                setValue={(source) => setClaimSource(source)}
+              />
+              <Spacing left={'small'} />
+              <DateTimePicker
+                disabled={createClaimLoading}
+                date={claimDate}
+                fullWidth={true}
+                setDate={(date) => setClaimDate(date)}
+                placeholder="Notification date"
+                maxDate={new Date()}
+              />
+              <Spacing left={'small'} />
+              <Button
+                disabled={createClaimLoading}
+                variation="danger"
+                color="danger"
+                onClick={() => {
+                  setShowForm(false)
+                  setClaimDate(null)
+                  setClaimSource(null)
+                }}
+              >
+                Cancel
+              </Button>
+              <Spacing left={'small'} />
+              <Button
+                disabled={
+                  claimSource === null ||
+                  claimDate === null ||
+                  createClaimLoading
+                }
+                variation="success"
+                color="success"
+                onClick={() => {
+                  if (claimSource === null || claimDate === null) {
+                    return
+                  }
+
+                  createClaim({
+                    variables: {
+                      memberId,
+                      date: format(claimDate, "yyyy-MM-dd'T'HH:mm:ss"),
+                      source: claimSource,
+                    },
+                  }).then((response) => {
+                    const claimId = response.data?.createClaim
+                    setShowForm(false)
+                    setClaimDate(null)
+                    setClaimSource(null)
+                    if (claimId) {
+                      history.push(`/claims/${claimId}/members/${memberId}`)
+                    }
+                    refetch()
```

Comment:
Guess we can skip some steps if we get the claim id? With that said don't we always get the claim Id?
```suggestion
                    const claimId = response.data?.createClaim
                    if (claimId) {
                      history.push(`/claims/${claimId}/members/${memberId}`)
                      return
                    }
                    setShowForm(false)
                    setClaimDate(null)
                    setClaimSource(null)
                    refetch()
```

Reply:
Do we even want the automatic navigation btw? Seems a bit weird to just fly away from the page, especially if it's not a 100% behaviour